Tests failed on master

```bash
$ sh run_tests.sh
```

```
testmeili
758e874a9ab7fe96ae6c034c7ef6e2032693c4c048a7b6756aa886e84a07271e
=== RUN   TestClientDocuments_Get
--- PASS: TestClientDocuments_Get (0.08s)
=== RUN   TestClientDocuments_Delete
--- PASS: TestClientDocuments_Delete (0.15s)
=== RUN   TestClientDocuments_Deletes
--- PASS: TestClientDocuments_Deletes (0.15s)
=== RUN   TestClientDocuments_List
    TestClientDocuments_List: client_documents_test.go:171: expected to return the document[1]
--- FAIL: TestClientDocuments_List (0.08s)
=== RUN   TestClientDocuments_AddOrReplace
    TestClientDocuments_AddOrReplace: client_documents_test.go:215: number of doc should be at least 1
--- FAIL: TestClientDocuments_AddOrReplace (0.08s)
=== RUN   TestClientDocuments_AddOrUpdate
    TestClientDocuments_AddOrUpdate: client_documents_test.go:259: number of doc should be at least 1
--- FAIL: TestClientDocuments_AddOrUpdate (0.09s)
=== RUN   TestClientDocuments_DeleteAllDocuments
--- PASS: TestClientDocuments_DeleteAllDocuments (0.09s)
=== RUN   TestClientHealth_Get
--- PASS: TestClientHealth_Get (0.00s)
=== RUN   TestClientHealth_Set
--- PASS: TestClientHealth_Set (0.01s)
=== RUN   TestClientIndexes_Create
--- PASS: TestClientIndexes_Create (0.02s)
=== RUN   TestClientIndexes_Get
--- PASS: TestClientIndexes_Get (0.02s)
=== RUN   TestClientIndexes_Delete
--- PASS: TestClientIndexes_Delete (0.03s)
=== RUN   TestClientIndexes_List
--- PASS: TestClientIndexes_List (0.07s)
=== RUN   TestClientIndexes_UpdateName
--- PASS: TestClientIndexes_UpdateName (0.02s)
=== RUN   TestClientIndexes_UpdatePrimaryKey
--- PASS: TestClientIndexes_UpdatePrimaryKey (0.02s)
=== RUN   TestClientKeys_Get
--- PASS: TestClientKeys_Get (0.00s)
=== RUN   TestClientSearch_Search
--- PASS: TestClientSearch_Search (0.33s)
=== RUN   TestClientSettings_GetAll
--- PASS: TestClientSettings_GetAll (0.02s)
=== RUN   TestClientSettings_UpdateAll
--- PASS: TestClientSettings_UpdateAll (0.08s)
=== RUN   TestClientSettings_ResetAll
--- PASS: TestClientSettings_ResetAll (0.08s)
=== RUN   TestClientSettings_GetRankingRules
--- PASS: TestClientSettings_GetRankingRules (0.02s)
=== RUN   TestClientSettings_UpdateRankingRules
--- PASS: TestClientSettings_UpdateRankingRules (0.09s)
=== RUN   TestClientSettings_ResetRankingRules
--- PASS: TestClientSettings_ResetRankingRules (0.08s)
=== RUN   TestClientSettings_GetDistinctAttribute
--- PASS: TestClientSettings_GetDistinctAttribute (0.02s)
=== RUN   TestClientSettings_UpdateDistinctAttribute
--- PASS: TestClientSettings_UpdateDistinctAttribute (0.08s)
=== RUN   TestClientSettings_ResetDistinctAttribute
--- PASS: TestClientSettings_ResetDistinctAttribute (0.08s)
=== RUN   TestClientSettings_GetSearchableAttributes
--- PASS: TestClientSettings_GetSearchableAttributes (0.02s)
=== RUN   TestClientSettings_UpdateSearchableAttributes
--- PASS: TestClientSettings_UpdateSearchableAttributes (0.08s)
=== RUN   TestClientSettings_ResetSearchableAttributes
--- PASS: TestClientSettings_ResetSearchableAttributes (0.08s)
=== RUN   TestClientSettings_GetDisplayedAttributes
--- PASS: TestClientSettings_GetDisplayedAttributes (0.02s)
=== RUN   TestClientSettings_UpdateDisplayedAttributes
--- PASS: TestClientSettings_UpdateDisplayedAttributes (0.09s)
=== RUN   TestClientSettings_ResetDisplayedAttributes
--- PASS: TestClientSettings_ResetDisplayedAttributes (0.09s)
=== RUN   TestClientSettings_GetStopWords
--- PASS: TestClientSettings_GetStopWords (0.02s)
=== RUN   TestClientSettings_UpdateStopWords
--- PASS: TestClientSettings_UpdateStopWords (0.08s)
=== RUN   TestClientSettings_ResetStopWords
--- PASS: TestClientSettings_ResetStopWords (0.08s)
=== RUN   TestClientSettings_GetSynonyms
--- PASS: TestClientSettings_GetSynonyms (0.02s)
=== RUN   TestClientSettings_UpdateSynonyms
--- PASS: TestClientSettings_UpdateSynonyms (0.08s)
=== RUN   TestClientSettings_ResetSynonyms
--- PASS: TestClientSettings_ResetSynonyms (0.08s)
=== RUN   TestClientSettings_GetAcceptNewFields
--- PASS: TestClientSettings_GetAcceptNewFields (0.02s)
=== RUN   TestClientSettings_UpdateAcceptNewFields
--- PASS: TestClientSettings_UpdateAcceptNewFields (0.08s)
=== RUN   TestClientSettings_GetAttributesForFaceting
--- PASS: TestClientSettings_GetAttributesForFaceting (0.02s)
=== RUN   TestClientSettings_UpdateAttributesForFaceting
--- PASS: TestClientSettings_UpdateAttributesForFaceting (0.08s)
=== RUN   TestClientSettings_ResetAttributesForFaceting
--- PASS: TestClientSettings_ResetAttributesForFaceting (0.14s)
=== RUN   TestClientStats_Get
--- PASS: TestClientStats_Get (0.07s)
=== RUN   TestClientUpdates_List
--- PASS: TestClientUpdates_List (0.02s)
=== RUN   TestClientVersion_Get
--- PASS: TestClientVersion_Get (0.00s)
=== RUN   Test_deleteAllIndexes
--- PASS: Test_deleteAllIndexes (0.24s)
FAIL
ok  	github.com/meilisearch/meilisearch-go	(cached)
```

## Tasks

- [x] Fix the code or the tests
- [ ] Why do the tests script returns `0` (= `SUCCEED`) when the tests fail? This should be fixed as well because it cannot currently be used like that in a CI

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -25,8 +25,11 @@ jobs:
             dep ensure
         fi
 
+    - name: Docker setup
+      run: docker run -d -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --master-key=masterKey --no-analytics=true
+
     - name: Test
-      run: sh run_tests.sh
+      run: go test -v ./...
 
     - name: Linter
       run: |

--- a/client_documents_test.go
+++ b/client_documents_test.go
@@ -7,11 +7,6 @@ import (
 func TestClientDocuments_Get(t *testing.T) {
 	var indexUID = "TestClientDocuments_Get"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -47,11 +42,6 @@ func TestClientDocuments_Get(t *testing.T) {
 
 func TestClientDocuments_Delete(t *testing.T) {
 	var indexUID = "TestClientDocuments_Delete"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -91,11 +81,6 @@ func TestClientDocuments_Delete(t *testing.T) {
 
 func TestClientDocuments_Deletes(t *testing.T) {
 	var indexUID = "deletes"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -137,11 +122,6 @@ func TestClientDocuments_Deletes(t *testing.T) {
 func TestClientDocuments_List(t *testing.T) {
 	var indexUID = "TestClientDocuments_List"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if _, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	}); err != nil {
@@ -178,11 +158,6 @@ func TestClientDocuments_List(t *testing.T) {
 
 func TestClientDocuments_AddOrReplace(t *testing.T) {
 	var indexUID = "TestClientDocuments_AddOrReplace"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -224,11 +199,6 @@ func TestClientDocuments_AddOrReplace(t *testing.T) {
 func TestClientDocuments_AddOrUpdate(t *testing.T) {
 	var indexUID = "TestClientDocuments_AddOrUpdate"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -268,11 +238,6 @@ func TestClientDocuments_AddOrUpdate(t *testing.T) {
 
 func TestClientDocuments_DeleteAllDocuments(t *testing.T) {
 	var indexUID = "TestClientDocuments_DeleteAllDocuments"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,

--- a/client_documents_test.go
+++ b/client_documents_test.go
@@ -8,7 +8,8 @@ func TestClientDocuments_Get(t *testing.T) {
 	var indexUID = "TestClientDocuments_Get"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -48,7 +49,8 @@ func TestClientDocuments_Delete(t *testing.T) {
 	var indexUID = "TestClientDocuments_Delete"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -91,7 +93,8 @@ func TestClientDocuments_Deletes(t *testing.T) {
 	var indexUID = "deletes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -135,7 +138,8 @@ func TestClientDocuments_List(t *testing.T) {
 	var indexUID = "TestClientDocuments_List"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if _, err := client.Indexes().Create(CreateIndexRequest{
@@ -176,7 +180,8 @@ func TestClientDocuments_AddOrReplace(t *testing.T) {
 	var indexUID = "TestClientDocuments_AddOrReplace"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -220,7 +225,8 @@ func TestClientDocuments_AddOrUpdate(t *testing.T) {
 	var indexUID = "TestClientDocuments_AddOrUpdate"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -264,7 +270,8 @@ func TestClientDocuments_DeleteAllDocuments(t *testing.T) {
 	var indexUID = "TestClientDocuments_DeleteAllDocuments"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{

--- a/client_health_test.go
+++ b/client_health_test.go
@@ -3,22 +3,12 @@ package meilisearch
 import "testing"
 
 func TestClientHealth_Get(t *testing.T) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if err := client.Health().Get(); err != nil {
 		t.Fatal(err)
 	}
 }
 
 func TestClientHealth_Set(t *testing.T) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if err := client.Health().Update(true); err != nil {
 		t.Fatal(err)
 	}

--- a/client_health_test.go
+++ b/client_health_test.go
@@ -4,7 +4,8 @@ import "testing"
 
 func TestClientHealth_Get(t *testing.T) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if err := client.Health().Get(); err != nil {
@@ -14,7 +15,8 @@ func TestClientHealth_Get(t *testing.T) {
 
 func TestClientHealth_Set(t *testing.T) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if err := client.Health().Update(true); err != nil {

--- a/client_indexes_test.go
+++ b/client_indexes_test.go
@@ -8,7 +8,8 @@ func TestClientIndexes_Create(t *testing.T) {
 	var indexUID = "TestClientIndexes_Create"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
@@ -28,7 +29,8 @@ func TestClientIndexes_Get(t *testing.T) {
 	var indexUID = "TestClientIndexes_Get"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
@@ -53,7 +55,8 @@ func TestClientIndexes_Delete(t *testing.T) {
 	var indexUID = "TestClientIndexes_Delete"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
@@ -77,7 +80,8 @@ func TestClientIndexes_List(t *testing.T) {
 	var indexUID = "TestClientIndexes_List"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -102,7 +106,8 @@ func TestClientIndexes_UpdateName(t *testing.T) {
 	var indexUID = "TestClientIndexes_UpdateName"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
@@ -127,7 +132,8 @@ func TestClientIndexes_UpdatePrimaryKey(t *testing.T) {
 	var indexUID = "TestClientIndexes_UpdatePrimaryKey"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{

--- a/client_indexes_test.go
+++ b/client_indexes_test.go
@@ -7,11 +7,6 @@ import (
 func TestClientIndexes_Create(t *testing.T) {
 	var indexUID = "TestClientIndexes_Create"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	resp, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -27,11 +22,6 @@ func TestClientIndexes_Create(t *testing.T) {
 
 func TestClientIndexes_Get(t *testing.T) {
 	var indexUID = "TestClientIndexes_Get"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -54,11 +44,6 @@ func TestClientIndexes_Get(t *testing.T) {
 func TestClientIndexes_Delete(t *testing.T) {
 	var indexUID = "TestClientIndexes_Delete"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	resp, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -78,11 +63,6 @@ func TestClientIndexes_Delete(t *testing.T) {
 
 func TestClientIndexes_List(t *testing.T) {
 	var indexUID = "TestClientIndexes_List"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -105,11 +85,6 @@ func TestClientIndexes_List(t *testing.T) {
 func TestClientIndexes_UpdateName(t *testing.T) {
 	var indexUID = "TestClientIndexes_UpdateName"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	resp, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -130,11 +105,6 @@ func TestClientIndexes_UpdateName(t *testing.T) {
 
 func TestClientIndexes_UpdatePrimaryKey(t *testing.T) {
 	var indexUID = "TestClientIndexes_UpdatePrimaryKey"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	resp, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,

--- a/client_keys_test.go
+++ b/client_keys_test.go
@@ -5,11 +5,6 @@ import (
 )
 
 func TestClientKeys_Get(t *testing.T) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if _, err := client.Keys().Get(); err != nil {
 		t.Fatal(err)
 	}

--- a/client_keys_test.go
+++ b/client_keys_test.go
@@ -6,7 +6,8 @@ import (
 
 func TestClientKeys_Get(t *testing.T) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if _, err := client.Keys().Get(); err != nil {

--- a/client_search_test.go
+++ b/client_search_test.go
@@ -8,11 +8,6 @@ import (
 func TestClientSearch_Search(t *testing.T) {
 	var indexUID = "TestClientSearch_Search"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})

--- a/client_search_test.go
+++ b/client_search_test.go
@@ -9,7 +9,8 @@ func TestClientSearch_Search(t *testing.T) {
 	var indexUID = "TestClientSearch_Search"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{

--- a/client_settings_test.go
+++ b/client_settings_test.go
@@ -9,11 +9,6 @@ import (
 func TestClientSettings_GetAll(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAll"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -47,11 +42,6 @@ func TestClientSettings_GetAll(t *testing.T) {
 func TestClientSettings_UpdateAll(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAll"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -84,11 +74,6 @@ func TestClientSettings_UpdateAll(t *testing.T) {
 func TestClientSettings_ResetAll(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetAll"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -109,11 +94,6 @@ func TestClientSettings_ResetAll(t *testing.T) {
 
 func TestClientSettings_GetRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_GetRankingRules"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -139,11 +119,6 @@ func TestClientSettings_GetRankingRules(t *testing.T) {
 func TestClientSettings_UpdateRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateRankingRules"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -166,11 +141,6 @@ func TestClientSettings_UpdateRankingRules(t *testing.T) {
 func TestClientSettings_ResetRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetRankingRules"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -190,11 +160,6 @@ func TestClientSettings_ResetRankingRules(t *testing.T) {
 
 func TestClientSettings_GetDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_GetDistinctAttribute"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -218,11 +183,6 @@ func TestClientSettings_GetDistinctAttribute(t *testing.T) {
 func TestClientSettings_UpdateDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateDistinctAttribute"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -244,11 +204,6 @@ func TestClientSettings_UpdateDistinctAttribute(t *testing.T) {
 func TestClientSettings_ResetDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetDistinctAttribute"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -269,11 +224,6 @@ func TestClientSettings_ResetDistinctAttribute(t *testing.T) {
 
 func TestClientSettings_GetSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_GetSearchableAttributes"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -297,11 +247,6 @@ func TestClientSettings_GetSearchableAttributes(t *testing.T) {
 func TestClientSettings_UpdateSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateSearchableAttributes"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -324,11 +269,6 @@ func TestClientSettings_UpdateSearchableAttributes(t *testing.T) {
 func TestClientSettings_ResetSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetSearchableAttributes"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -348,11 +288,6 @@ func TestClientSettings_ResetSearchableAttributes(t *testing.T) {
 
 func TestClientSettings_GetDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_GetDisplayedAttributes"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -376,11 +311,6 @@ func TestClientSettings_GetDisplayedAttributes(t *testing.T) {
 func TestClientSettings_UpdateDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateDisplayedAttributes"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -403,11 +333,6 @@ func TestClientSettings_UpdateDisplayedAttributes(t *testing.T) {
 func TestClientSettings_ResetDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetDisplayedAttributes"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -427,11 +352,6 @@ func TestClientSettings_ResetDisplayedAttributes(t *testing.T) {
 
 func TestClientSettings_GetStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_GetStopWords"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -456,11 +376,6 @@ func TestClientSettings_GetStopWords(t *testing.T) {
 func TestClientSettings_UpdateStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateStopWords"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -484,11 +399,6 @@ func TestClientSettings_UpdateStopWords(t *testing.T) {
 func TestClientSettings_ResetStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetStopWords"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -509,11 +419,6 @@ func TestClientSettings_ResetStopWords(t *testing.T) {
 
 func TestClientSettings_GetSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_GetSynonyms"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -537,11 +442,6 @@ func TestClientSettings_GetSynonyms(t *testing.T) {
 
 func TestClientSettings_UpdateSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateSynonyms"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -567,11 +467,6 @@ func TestClientSettings_UpdateSynonyms(t *testing.T) {
 func TestClientSettings_ResetSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetSynonyms"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -592,11 +487,6 @@ func TestClientSettings_ResetSynonyms(t *testing.T) {
 
 func TestClientSettings_GetAcceptNewFields(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAcceptNewFields"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -620,11 +510,6 @@ func TestClientSettings_GetAcceptNewFields(t *testing.T) {
 func TestClientSettings_UpdateAcceptNewFields(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAcceptNewFields"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})
@@ -645,11 +530,6 @@ func TestClientSettings_UpdateAcceptNewFields(t *testing.T) {
 
 func TestClientSettings_GetAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAttributesForFaceting"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
@@ -672,11 +552,6 @@ func TestClientSettings_GetAttributesForFaceting(t *testing.T) {
 
 func TestClientSettings_UpdateAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAttributesForFaceting"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	attributesForFaceting := []string{"tag", "title"}
 
@@ -704,11 +579,6 @@ func TestClientSettings_UpdateAttributesForFaceting(t *testing.T) {
 
 func TestClientSettings_ResetAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetAttributesForFaceting"
-
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 
 	attributesForFaceting := []string{"tag", "title"}
 

--- a/client_settings_test.go
+++ b/client_settings_test.go
@@ -10,7 +10,8 @@ func TestClientSettings_GetAll(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAll"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -47,7 +48,8 @@ func TestClientSettings_UpdateAll(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAll"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -83,7 +85,8 @@ func TestClientSettings_ResetAll(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetAll"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -108,7 +111,8 @@ func TestClientSettings_GetRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_GetRankingRules"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -136,7 +140,8 @@ func TestClientSettings_UpdateRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateRankingRules"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -162,7 +167,8 @@ func TestClientSettings_ResetRankingRules(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetRankingRules"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -186,7 +192,8 @@ func TestClientSettings_GetDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_GetDistinctAttribute"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -212,7 +219,8 @@ func TestClientSettings_UpdateDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateDistinctAttribute"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -237,7 +245,8 @@ func TestClientSettings_ResetDistinctAttribute(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetDistinctAttribute"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -262,7 +271,8 @@ func TestClientSettings_GetSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_GetSearchableAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -288,7 +298,8 @@ func TestClientSettings_UpdateSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateSearchableAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -314,7 +325,8 @@ func TestClientSettings_ResetSearchableAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetSearchableAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -338,7 +350,8 @@ func TestClientSettings_GetDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_GetDisplayedAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -364,7 +377,8 @@ func TestClientSettings_UpdateDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateDisplayedAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -390,7 +404,8 @@ func TestClientSettings_ResetDisplayedAttributes(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetDisplayedAttributes"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -414,7 +429,8 @@ func TestClientSettings_GetStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_GetStopWords"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -441,7 +457,8 @@ func TestClientSettings_UpdateStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateStopWords"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -468,7 +485,8 @@ func TestClientSettings_ResetStopWords(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetStopWords"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -493,7 +511,8 @@ func TestClientSettings_GetSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_GetSynonyms"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -520,7 +539,8 @@ func TestClientSettings_UpdateSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateSynonyms"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -548,7 +568,8 @@ func TestClientSettings_ResetSynonyms(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetSynonyms"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -573,7 +594,8 @@ func TestClientSettings_GetAcceptNewFields(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAcceptNewFields"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -599,7 +621,8 @@ func TestClientSettings_UpdateAcceptNewFields(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAcceptNewFields"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -624,7 +647,8 @@ func TestClientSettings_GetAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_GetAttributesForFaceting"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{
@@ -650,7 +674,8 @@ func TestClientSettings_UpdateAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_UpdateAttributesForFaceting"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	attributesForFaceting := []string{"tag", "title"}
@@ -681,7 +706,8 @@ func TestClientSettings_ResetAttributesForFaceting(t *testing.T) {
 	var indexUID = "TestClientSettings_ResetAttributesForFaceting"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	attributesForFaceting := []string{"tag", "title"}

--- a/client_stats_test.go
+++ b/client_stats_test.go
@@ -6,7 +6,8 @@ import (
 
 func TestClientStats_Get(t *testing.T) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if _, err := client.Stats().GetAll(); err != nil {

--- a/client_stats_test.go
+++ b/client_stats_test.go
@@ -5,11 +5,6 @@ import (
 )
 
 func TestClientStats_Get(t *testing.T) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if _, err := client.Stats().GetAll(); err != nil {
 		t.Fatal(err)
 	}

--- a/client_updates_test.go
+++ b/client_updates_test.go
@@ -8,7 +8,8 @@ func TestClientUpdates_List(t *testing.T) {
 	var indexUID = "TestClientUpdates_List"
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	_, err := client.Indexes().Create(CreateIndexRequest{

--- a/client_updates_test.go
+++ b/client_updates_test.go
@@ -7,11 +7,6 @@ import (
 func TestClientUpdates_List(t *testing.T) {
 	var indexUID = "TestClientUpdates_List"
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	_, err := client.Indexes().Create(CreateIndexRequest{
 		UID: indexUID,
 	})

--- a/client_version_test.go
+++ b/client_version_test.go
@@ -5,11 +5,6 @@ import (
 )
 
 func TestClientVersion_Get(t *testing.T) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	if _, err := client.Version().Get(); err != nil {
 		t.Fatal(err)
 	}

--- a/client_version_test.go
+++ b/client_version_test.go
@@ -6,7 +6,8 @@ import (
 
 func TestClientVersion_Get(t *testing.T) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	if _, err := client.Version().Get(); err != nil {

--- a/main_test.go
+++ b/main_test.go
@@ -32,12 +32,13 @@ func deleteAllIndexes(client *Client) (ok bool, err error) {
 
 func TestMain(m *testing.M) {
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 	deleteAllIndexes(client)
-	m.Run()
+	code := m.Run()
 	deleteAllIndexes(client)
-	os.Exit(0)
+	os.Exit(code)
 }
 
 func Test_deleteAllIndexes(t *testing.T) {
@@ -48,7 +49,8 @@ func Test_deleteAllIndexes(t *testing.T) {
 	}
 
 	var client = NewClient(Config{
-		Host: "http://localhost:7700",
+		Host:   "http://localhost:7700",
+		APIKey: "masterKey",
 	})
 
 	for _, uid := range indexUIDs {
@@ -57,7 +59,7 @@ func Test_deleteAllIndexes(t *testing.T) {
 		})
 
 		if err != nil {
-			t.Fatal(err)
+			t.Error(err)
 		}
 	}
 
@@ -66,10 +68,10 @@ func Test_deleteAllIndexes(t *testing.T) {
 	for _, uid := range indexUIDs {
 		resp, err := client.Indexes().Get(uid)
 		if err == nil {
-			t.Fatal(err)
+			t.Error(err)
 		}
 		if resp != nil {
-			t.Fatal("deleteAllIndexes: One or more indexes were not deleted")
+			t.Error("deleteAllIndexes: One or more indexes were not deleted")
 		}
 	}
 }

--- a/main_test.go
+++ b/main_test.go
@@ -30,11 +30,12 @@ func deleteAllIndexes(client *Client) (ok bool, err error) {
 	return true, nil
 }
 
+var client = NewClient(Config{
+	Host:   "http://localhost:7700",
+	APIKey: "masterKey",
+})
+
 func TestMain(m *testing.M) {
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
 	deleteAllIndexes(client)
 	code := m.Run()
 	deleteAllIndexes(client)
@@ -48,18 +49,13 @@ func Test_deleteAllIndexes(t *testing.T) {
 		"Test_deleteAllIndexes3",
 	}
 
-	var client = NewClient(Config{
-		Host:   "http://localhost:7700",
-		APIKey: "masterKey",
-	})
-
 	for _, uid := range indexUIDs {
 		_, err := client.Indexes().Create(CreateIndexRequest{
 			UID: uid,
 		})
 
 		if err != nil {
-			t.Error(err)
+			t.Fatal(err)
 		}
 	}
 
@@ -68,10 +64,10 @@ func Test_deleteAllIndexes(t *testing.T) {
 	for _, uid := range indexUIDs {
 		resp, err := client.Indexes().Get(uid)
 		if err == nil {
-			t.Error(err)
+			t.Fatal(err)
 		}
 		if resp != nil {
-			t.Error("deleteAllIndexes: One or more indexes were not deleted")
+			t.Fatal("deleteAllIndexes: One or more indexes were not deleted")
 		}
 	}
 }

--- a/run_tests.sh
+++ b/run_tests.sh
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 docker kill testmeili \
-; docker run -d -p 7700:7700 --name testmeili -e MEILI_NO_ANALYTICS=1 --rm getmeili/meilisearch:latest\
+; docker run -d -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --master-key=masterKey --no-analytics=true\
 ; sleep 1 \
-; go test -v ./...
+; go test -v -count=1 ./...

